### PR TITLE
New schema for selection indicator ref data

### PR DESCRIPTION
**Pull request by JimmySunDance**

### CODE REVIEW

File: schemas/reference/V60__selectionindicator.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+CREATE TABLE selectionindicator(
+  id INTEGER NOT NULL PRIMARY KEY, 
+  mode VARCHAR NOT NULL REFERENCES modeoftransport(id),
+  indicator VARCHAR NOT NULL,
+  validfrom TIMESTAMP WITH TIME ZONE,
+  validto TIMESTAMP WITH TIME ZONE
+);
+
+-- Table comment
+COMMENT ON TABLE selectionindicator IS '{"label": "Selection Indicator", "description": "A list of reasons for conducting a search on a particular vehical.", "schemalastupdated": "15/07/2019", "dataversion": 1}';
```
Comment (Ben-UK) [inline schemas/reference/V60__selectionindicator.sql]: *vehicle

File: schemas/reference/V60__selectionindicator.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
+CREATE TABLE selectionindicator(
+  id INTEGER NOT NULL PRIMARY KEY, 
+  mode VARCHAR NOT NULL REFERENCES modeoftransport(id),
```
Comment (Ben-UK) [inline schemas/reference/V60__selectionindicator.sql]: Should the data type be INT4?

File: schemas/reference/V60__selectionindicator.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  validfrom TIMESTAMP WITH TIME ZONE,
+  validto TIMESTAMP WITH TIME ZONE
+);
+
+-- Table comment
+COMMENT ON TABLE selectionindicator IS '{"label": "Selection Indicator", "description": "A list of reasons for conducting a search on a particular vehical.", "schemalastupdated": "15/07/2019", "dataversion": 1}';
+-- Column comments
+COMMENT ON COLUMN selectionindicator.id IS '{"label": "Identifier", "description": "Database unique identity record.", "summaryview": "false"}';
+COMMENT ON COLUMN selectionindicator.mode IS '{"label": "Mode of transport", "description": "The mode of transport used.", "summaryview": "true"}';
+COMMENT ON COLUMN selectionindicator.indicator IS '{"description": "Reason for selecting vehical for search.", "summaryview": "true"}';
```
Comment (Ben-UK) [inline schemas/reference/V60__selectionindicator.sql]: *vehicle

File: schemas/reference/V60__selectionindicator.sql [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
```
Comment (tomfitzherbert) [inline schemas/reference/V60__selectionindicator.sql]: Missing the `label`

File: schemas/reference/V60__selectionindicator.sql (hunk 2 above)
Comment (tomfitzherbert) [reply schemas/reference/V60__selectionindicator.sql]: I think that `INTEGER` is fine, and is consistent with other tables

Comment (JimmySunDance) [reply schemas/reference/V60__selectionindicator.sql]: fixed

File: schemas/reference/V60__selectionindicator.sql (hunk 3 above)
Comment (JimmySunDance) [reply schemas/reference/V60__selectionindicator.sql]: fixed

File: schemas/reference/V60__selectionindicator.sql (hunk 1 above)
Comment (JimmySunDance) [reply schemas/reference/V60__selectionindicator.sql]: fixed

**Pull request merged**